File: java/src/main/java/com/fish/array/Q2445.java
package com.fish.array;

/**
 * @author liuqi
 * @date 2023/5/29
 */
public class Q2445 {
    class Solution {
        public int averageValue(int[] nums) {
            int sum = 0;
            int cnt = 0;
            for (int num : nums) {
                if (num % 6 == 0) {
                    cnt++;
                    sum += num;
                }
            }
            return cnt == 0 ? 0 : sum / cnt;
        }
    }
}

File: python/Q2445.py
from typing import List


class Solution:
    def averageValue(self, nums: List[int]) -> int:
        cnt = 0
        sum = 0
        for num in nums:
            if num % 6 == 0:
                cnt += 1
                sum += num
        return 0 if cnt==0 else sum//cnt